src/version.h

```diff
@@ -4,15 +4,58 @@
 #ifndef BITCOIN_VERSION_H
 #define BITCOIN_VERSION_H
 
-#include "clientversion.h"
+#if defined(HAVE_CONFIG_H)
+#include "config/gridcoin-config.h"
+#else
+
+//! Copyright string used in Windows .rc files
+#define COPYRIGHT_STR "2009-"
+
+//These need to be macros, as version.cpp's and bitcoin-qt.rc's voodoo requires it
+#define CLIENT_VERSION_MAJOR       3
+#define CLIENT_VERSION_MINOR       6
+#define CLIENT_VERSION_REVISION    3
+#define CLIENT_VERSION_BUILD       17
+
+#define CLIENT_VERSION_IS_RELEASE false
+#define COPYRIGHT_YEAR 2017
+
+#endif //HAVE_CONFIG_H
+
+// Converts the parameter X to a string after macro replacement on X has been performed.
+// Don't merge these into one macro!
+#define STRINGIZE(X) DO_STRINGIZE(X)
+#define DO_STRINGIZE(X) #X
+
+#define COPYRIGHT_STR "2009-" STRINGIZE(COPYRIGHT_YEAR) " The Bitcoin Core Developers"
```

Comment:
and this is going to conflict with the definition from above

Reply:
Agreed, however this line is the correct definition (syntactically). The content of the string should be talked about though. How about:
"2009-" STRINGIZE(COPYRIGHT_YEAR) " The Bitcoin Core and Gridcoin Developers"